documentation/ipi-install/modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -109,4 +109,4 @@ Use the following steps to install a container that contains the images.
 registry.centos.org/centos/httpd-24-centos7:latest
 ----
 
-. The above command creates a caching webserver with the name `rhcos_image_cache` which will be serving the images for deployment. The first image `${RHCOS_PATH}${RHCOS_QEMU_URI}` will be used as `bootstrapOSImage` and the second image `${RHCOS_PATH}${RHCOS_OPENSTACK_URI}` will be used as `clusterOSImage` in `install-config.yaml` file as shown in xref:additional-install-config-parameters[Additional install-config parameters] section. 
+. The above command creates a caching webserver with the name `rhcos_image_cache` which will be serving the images for deployment. The first image `${RHCOS_PATH}${RHCOS_QEMU_URI}` will be used as `bootstrapOSImage` and the second image `${RHCOS_PATH}${RHCOS_OPENSTACK_URI}` will be used as `clusterOSImage` in `install-config.yaml` file as shown in xref:additional-install-config-parameters_{context}[Additional install-config parameters] section. 
```

Comment:
```
${RHCOS_PATH}${RHCOS_QEMU_URI}
${RHCOS_PATH}${RHCOS_OPENSTACK_URI}
```
The above is not accurate. If these values get expanded and you compare what this provides versus what needs to go into the install-config.yaml for `boostrapOSImage` and `clusterOSImage` you will see it wouldn't match a live example.